### ISSUE TITLE
Reason for failing Travis/CI Jobs

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/docker-formula#182]
The failures are caused by following line in containerd.service file. Its OS packaging issue.

`
ExecStartPre=/sbin/modprobe overlay
`

See: https://github.com/docker/for-linux/issues/475#issuecomment-437415452 
For now they can be ignored.

UPDATE: Now failing with `       [ERROR   ] The renderer "yaml_jinja" is not available`

**Comment by noelmcloughlin** [comment]
The Travis CI jobs are now failing with this error.  Any ideas @javierbertoli @aboe76 
```
       [ERROR   ] The renderer "yaml_jinja" is not available
```

**Comment by noelmcloughlin** [comment]
I tried to fix this in #213 without success. This formula should be migrated to inspec.

**Comment by noelmcloughlin** [comment]
See previous comment.
